src/__tests__/resources/assetMappings3D.integration.spec.ts

```diff
@@ -0,0 +1,158 @@
+
+  test('create file', async () => {
+    const fileContent = readFileSync('src/__tests__/test3dFile.fbx');
+    const fileProps = { name: `file_revision_test_${now}.fbx` };
+    file = await client.files.upload(fileProps, fileContent);
+    expect(file).toBeTruthy();
+  });
+
+  test(
+    'create revision',
+    async done => {
+      const revisionsToCreate: CreateRevision3D[] = [{ fileId: file.id }];
+      revisions = await retryInSeconds(
+        () => client.revisions3D.create(model.id, revisionsToCreate),
+        3,
+        400,
+        60
+      );
+      expect(revisions.length).toBe(1);
+      done();
+    },
+    10 * 1000
+  );
+
+  const rootAsset = {
+    name: 'test-root',
+    description: 'Root asset for cognitesdk-js test',
+    metadata: {
+      refId: 'test-root',
+    },
+    refId: 'test-root',
+  };
+
+  const childAsset = {
+    name: 'test-child',
+    description: 'Child asset for cognitesdk-js test',
+    metadata: {
+      refId: 'test-child',
+    },
+    parentRefId: 'test-root',
+  };
+
+  test('create assets', async () => {
+    assets = await client.assets.create([childAsset, rootAsset]);
+  });
+
+  let nodes3D: Node3D[];
+  test(
+    'list 3d nodes',
+    async done => {
+      const req = async () =>
+        client.revisions3D
+          .list3DNodes(model.id, revisions[0].id)
+          .autoPagingToArray();
+      nodes3D = await retryInSeconds(req, 5, 404, 5 * 60);
+      expect(nodes3D.map(n => n.name)).toContain('input_Input_1.fbx');
+      done();
+    },
+    5 * 60 * 1000
+  );
+
+  let assetMappingsToCreate: CreateAssetMapping3D[];
+  test('create asset mappings 3d', async () => {
+    assetMappingsToCreate = nodes3D.slice(0, 2).map((node, index) => ({
+      nodeId: node.id,
+      assetId: Number(assets[index].id),
+    }));
+    assetMappings = await client.assetMappings3D.create(
+      model.id,
+      revisions[0].id,
+      assetMappingsToCreate
+    );
+    expect(assetMappings.map(a => a.assetId).sort(simpleCompare)).toEqual(
+      assetMappingsToCreate.map(a => a.assetId).sort(simpleCompare)
+    );
+    expect(assetMappings.length).toBe(2);
+  });
+
+  test('list asset mappings 3d', async () => {
+    const listed = await client.assetMappings3D
+      .list(model.id, revisions[0].id)
+      .autoPagingToArray({ limit: 2 });
+    expect(listed.map(m => m.assetId).sort(simpleCompare)).toEqual(
+      assetMappingsToCreate.map(m => m.assetId).sort(simpleCompare)
+    );
+    expect(listed.map(m => m.nodeId).sort(simpleCompare)).toEqual(
+      assetMappingsToCreate.map(m => m.nodeId).sort(simpleCompare)
+    );
+    expect(listed.length).toBe(2);
+  });
+
+  test('delete asset mappings 3d', async () => {
+    const deleted = await client.assetMappings3D.delete(
+      model.id,
+      revisions[0].id,
+      assetMappingsToCreate
+    );
+    expect(deleted).toEqual({});
+  });
+
+  test('list asset mappings 3d (empty)', async () => {
+    const listed = await client.assetMappings3D
```

Comment:
`listed` not `list`?
Why is it empty? Unclear to me

Reply:
because I called `delete` above
this test checks if they have been actually deleted

Reply:
I see